### DISCUSSION TITLE
Antler Engraving

### DISCUSSION THREAD

**Discussion by Steph152** [arkypita/LaserGRBL#1682] [Q&A]
Hello! I have a 5w Atomstack engraver. I’m trying to engrave some writing on an antler. So far all I’m getting is burnt antler. Does anyone have any recommendations for settings? Do I need to do something to prep the antler? Thanks! :)
![842D5929-7D0F-4803-A421-31767DFBEEF8](https://user-images.githubusercontent.com/92003818/146661315-23e596f9-f034-4407-ba1a-6905c133ff55.jpeg)

**Comment by Tribo-2dot0** [comment]
you should probably decrease the laser power and/or increase the speed...
Test this pattern to better determine
![Laser-Gravur](https://user-images.githubusercontent.com/83982525/146812312-36b6eab9-839b-4f7b-bf52-04dc7dff6dfc.png)
:
